packages/eas-cli/src/build/metadata.ts

```diff
@@ -26,5 +27,8 @@ export function collectMetadata<T extends Platform>(
     sdkVersion: ctx.commandCtx.exp.sdkVersion,
     trackingContext: ctx.trackingCtx,
     distribution: ctx.buildProfile.distribution ?? DistributionType.STORE,
+    appName: ctx.commandCtx.exp.name,
+    appIdentifier:
+      (await getAppIdentifierAsync(ctx.commandCtx.projectDir, ctx.platform)) ?? undefined,
```

Comment:
Not related to this PR, but we probably need to refactor everything from projectUtils to take config (or manifest) instead of projectDir